custom/icds_reports/tasks.py

```diff
@@ -1179,3 +1179,36 @@ def _bust_awc_cache():
     for key in reach_keys:
         cache.delete(key)
     create_datadog_event('redis: delete dashboard keys', 'finish')
+
+
+@task(queue='dashboard_comparison_queue')
+def run_citus_experiment_raw_sql(parameterized_sql, params, data_source="Unknown"):
```

Comment:
do any date params need to be deserialised?

Reply:
In my limited testing, not deserializng them should work fine because they are parameters to the queries and not trying to inspect them as dates. Strings in execute should be functionally the same as dates.